waterloop-site/src/theme/global.scss

```diff
@@ -697,10 +697,14 @@ h1 {
     font-family: IBM Plex Sans;
     font-style: normal;
```

Comment:
Change font style to italic and add text-transform: none to match the Figma header styling.

Reply:
I had previously used 1.5rem for the SocialMediaWrapper and RoutingWrapper components used in the footer. Should we change the font size there too?